com.github.muriloventuroso.pdftricks.json

```diff
@@ -11,37 +11,33 @@
         "--socket=x11",
         "--device=dri",
         "--socket=wayland",
-        "--filesystem=home",
         "--talk-name=ca.desrt.dconf",
-        "--env=DCONF_USER_CONFIG_DIR=.config/dconf",
```

Comment:
Those are needed if you gsettings. If not, you need to remove the rest of dconf permissions too.